lsp-metals.el

```diff
@@ -173,11 +173,27 @@ Should be ignored if there is no open doctor window."
     (overlay-put ov 'help-echo hover)
     (push ov lsp-metals--decorations)))
 
+(defun lsp-metals--logs-toggle (_workspace)
+  "Focus or remove focus on the output logs reported by the
+server via `window/logMessage'."
+  (switch-to-buffer (get-buffer-create "*lsp-log*")))
+
+(defun lsp-metals--diagnostics-focus (_workspace)
+  "Focus on the window that lists all published diagnostics."
+  (defvar flymake-mode)
+  (defvar flycheck-mode)
+  (when flymake-mode
```

Comment:
maybe it makes sense to include `lsp-treemacs-errors-list` as well.

Reply:
Added check for `lsp-treemacs-errors-list`.